railties/lib/rails/tasks/dev.rake

```diff
@@ -1,14 +1,16 @@
 namespace :dev do
   desc 'Toggle development mode caching on/off'
   task :cache do
+    FileUtils.mkdir_p('tmp')
+  
     if File.exist? 'tmp/caching-dev.txt'
       File.delete 'tmp/caching-dev.txt'
       puts 'Development mode is no longer being cached.'
     else
       FileUtils.touch 'tmp/caching-dev.txt'
       puts 'Development mode is now being cached.'
-    end
-
+    end 
```

Comment:
Could you please remove the extra whitespace that there is in the end of this line and the line below?